### ISSUE TITLE
Assigning role to a user

### ISSUE THREAD

**Issue by sonu3706** [okta/okta-sdk-java#249]
I am using Okta SDK along with Spring boot. I created User in okta along temporary password and now I want to assign a specific role to a user like ADMIN, USER etc.
But as I am not able to find out any way to do this with SDK.
And there is no documentation for the same in okta developer.
 I will tell you what I did :-1: 
1. Created user account in okta 
```java
 TempPassword oktaAdmin = UserBuilder.instance().setFirstName(tenantUser.getFirstName())
		.setLastName(tenantUser.getLastName())
		.setEmail(tenantUser.getEmail())
		.setLogin(tenantUser.getEmail())
		.setPassword(password.trim().toCharArray())
		.setMobilePhone(tenantUser.getUserMobilePhone())
		.putProfileProperty("tenantId", tenantUser.getTurboTenant().getName())
		.buildAndCreate(client)
		.expirePassword();
```
and  then 
by `User user = client.getUser(tenantUser.getUserName()).getId();`
I am fetching ID for the user.

2. Created Role class and instantiated it  
 `Role role = client.instantiate(Role.class);`
after step 2 I am clueless in how to create and assign `Admin` role to created user

**Comment by sonu3706** [comment]
@bdemers Thanks for the suggestions. Is there any other way we can achieve this functionality using SDK??

**Comment by bdemers** [comment]
You could also do it with little casting to the implementation of `DefaultRole` and call `put("type", "your-role-type")`, NOTE: this requires changing the dependency scope of the `okta-sdk-impl` to `compile`, which we generally recommend against as this module's implementation details are subject to change.
That said, `put`/`get` is likely going to be exposed on all of our resources in the next release.

**Comment by bdemers** [comment]
Resolved in 1.4.0
